storybookjs/storybook
# fix for when outputDir is an absolute path

Issue:  Absolute path output in build-storybook doesn't work as expected #5334

## What I did
Added a check for `outputDir` to see if its a absolute path already before trying to resolve it into one.
## How to test
Set `--output-path` to an absolute path

- Is this testable with Jest or Chromatic screenshots? not that I know of
- Does this need a new example in the kitchen sink apps? no
- Does this need an update to the documentation? no

If your answer is yes to any of these, please make sure to include it in your PR.
